tools/harness-automation/autothreadharness/harness_case.py

```diff
@@ -226,7 +227,13 @@ def _init_harness(self):
         harness_config.read('%s\\Config\\Configuration.ini' % settings.HARNESS_HOME)
         if harness_config.has_option('THREAD_HARNESS_CONFIG', 'BrowserAutoNavigate') and \
                 harness_config.getboolean('THREAD_HARNESS_CONFIG', 'BrowserAutoNavigate'):
-            os.system('taskkill /t /f /im chrome.exe')
+            logger.error('BrowserAutoNavigate in Configuration.ini should be False')
+            raise FailError('BrowserAutoNavigate in Configuration.ini should be False')
+        if settings.MIXED_DEVICE_TYPE:
+            if harness_config.has_option('THREAD_HARNESS_CONFIG', 'EnableDeviceSelection') and \
+                    not harness_config.getboolean('THREAD_HARNESS_CONFIG', 'EnableDeviceSelection'):
+                logger.error('EnableDeviceSelection in Configuration.ini should be True')
+                raise FailError('EnableDeviceSelection in Configuration.ini should be True')
```

Comment:
Please submit feedback on https://openthread.io/certification/automation_setup to add this configuration introduction for mixed testbed as well.

Reply:
Sure, I will file a feedback for IO